precompiles/dapps-staking/DappsStaking.sol

```diff
@@ -54,12 +54,16 @@ interface DappsStaking {
     /// @notice Withdraw all funds that have completed the unbonding process.
     function withdraw_unbonded() external;
 
-    /// @notice Claim one era of unclaimed staker rewards for the specifeid contract.
+    /// @notice Claim earned staker rewards for the oldest unclaimed era.
```

Comment:
Why not add a disclaimer on top that devs should read pallet rustdocs to better understand these functionalities?

Reply:
added  on top of the file